Documentation/Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.md

```diff
@@ -293,3 +293,84 @@ kubectl delete -f object-store.yaml
 ```
 
 Removing object store(s) from the master zone of the master zone group should be done with caution. When all of these object-stores are deleted the period cannot be updated and that realm cannot be pulled.
+
+#### Set up Multisite Configs for already existing CephObjectStore
+
+When a single site `CephObjectStore` CR request comes to Rook, it internally creates zone/zonegroup/realm with the same name as object store. So user can just export them into Kubernetes resources using the same name. For example consider following `CephObjectStore`
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: my-store
+  namespace: rook-ceph # namespace:cluster
+spec:
+  metadataPool:
+    replicated:
+      size: 1
+  dataPool:
+    replicated:
+      size: 1
+  preservePoolsOnDelete: false
+  gateway:
+    port: 80
+    instances: 1
+```
+
+For creating multisite related CRs use the following:
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephObjectRealm
+metadata:
+  name: my-store
+  namespace: rook-ceph # namespace:cluster
+---
+apiVersion: ceph.rook.io/v1
+kind: CephObjectZoneGroup
+metadata:
+  name: my-store
+  namespace: rook-ceph # namespace:cluster
+spec:
+  realm: my-store
+---
+apiVersion: ceph.rook.io/v1
+kind: CephObjectZone
+metadata:
+  name: my-store
+  namespace: rook-ceph # namespace:cluster
+spec:
+  zoneGroup: my-store
+  metadataPool:
+    replicated:
+      size: 1
+  dataPool:
+    replicated:
+      size: 1
+  preservePoolsOnDelete: false
+  # recommended to set this value if ingress used for exposing rgw endpoints
+  # customEndpoints:
+  #   - "http://rgw-a.fqdn"
+```
+After that modify the `CephObjectStore` CRD and apply below changes on it, this trigger creating `realm system user` secret for multisite.
```

Comment:
```suggestion
After that modify the existing `CephObjectStore` CR and apply below changes to trigger creation of the `realm system user` secret for multisite.
```

Reply:
Where is the `realm system user` secret found? Can we link to somewhere else in the docs for that?

Reply:
How about pointing out the changes to the object store CR? Is the only change that the pools are removed since they belong to the zone now?